tests/distributions/test_distributions.py

```diff
@@ -151,17 +152,22 @@ def test_expand_by(dist, sample_shape, shape_type):
 
 @pytest.mark.parametrize('sample_shape', [(), (2,), (2, 3)])
 @pytest.mark.parametrize('shape_type', [torch.Size, tuple, list])
-def test_expand_new_dim(dist, sample_shape, shape_type):
+@pytest.mark.parametrize('default', [False, True])
+def test_expand_new_dim(dist, sample_shape, shape_type, default):
     for idx in range(dist.get_num_test_data()):
         small = dist.pyro_dist(**dist.get_dist_params(idx))
         with xfail_if_not_implemented():
```

Comment:
Can we now remove any of the `xfail_if_not_implemented`s, here and in other tests below?

Reply:
Done - I have restricted the scope of `xfail` so that it only applies to custom implementations.